cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -33,21 +31,38 @@ DESCRIPTION
                Department department;
            |};
 
-       Then, generate the client API and SQL script based on the data model. Use the generated
-       API to query and manipulate the persistent data in the application and use the generated
-       SQL script to create the database tables.
+       Then, generate the client API based on the data model. You can use the `bal build` command to generate the client.
+       This process regenerates the client when modifying the data model during project builds.
+       Use the generated API to query and manipulate the persistent data in the application.
```

Comment:
```suggestion
       Then, generate the client API based on the data model. Use the `bal build` command to generate the client API.
       During project builds, this process updates the client API whenever modifications are made to the data model.
       Use the generated API to query and manipulate the persistent data in the application.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/commit/ef73dd6fdbd23ed185882b2a23767ffaa7c67725